app/views/publications/monthly_statistics/temporarily_unavailable.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= content_for :title, 'Monthly Statistics - ITT2023' %>
+
+<p class="govuk-body">
```

Comment:
Can we wrap the whole page in these HTML tags to limit the line width please (makes it easier to read):

```html
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">

  [...]  

  </div>
</div>
```

Reply:
Yep